.ci/install_td_shim.sh

```diff
@@ -17,7 +17,7 @@ main() {
 	build_static_artifact_and_install "tdx-td-shim"
 
 	if [ "${KATA_BUILD_CC:-no}" == "yes" ]; then
-		sudo ln -sf /opt/confidential-containers/share/td-shim \
+		sudo ln -sf /opt/kata/share/td-shim \
```

Comment:
I never understood those td-shim and tdvf symlinks in `/usr/share` . My opportunity to ask: why we need them?

Other than that, could be use `$PREFIX` here to make it configurable?

Reply:
Yes, it's fair IMO.

Yeah, have no idea why the symlink and it seems I will not know the answer anytime soon :(